src/components/cc-email-list/cc-email-list.stories.js

```diff
@@ -0,0 +1,424 @@
+import './cc-email-list.js';
+import './cc-email-list.smart.js';
+import { makeStory, storyWait } from '../../stories/lib/make-story.js';
+import { enhanceStoriesNames } from '../../stories/lib/story-names.js';
+
+const SAMPLE_EMAIL_ADDRESS = 'sample.email@clever-cloud.com';
+const ANOTHER_SAMPLE_EMAIL_ADDRESS = 'another.sample.email@clever-cloud.com';
+const YET_ANOTHER_SAMPLE_EMAIL_ADDRESS = 'yet.another.sample.email@clever-cloud.com';
+const HUGE_EMAIL_ADDRESS = `hugeemaila${'d'.repeat(500)}ress@clever-cloud.com`;
+
+const primaryAddress = { state: 'idle', address: SAMPLE_EMAIL_ADDRESS, verified: true };
+const primaryUnverified = { state: 'idle', address: SAMPLE_EMAIL_ADDRESS, verified: false };
+const secondaryAddresses = [
+  { state: 'idle', address: ANOTHER_SAMPLE_EMAIL_ADDRESS, verified: true },
+  { state: 'idle', address: YET_ANOTHER_SAMPLE_EMAIL_ADDRESS, verified: true },
+];
+const secondaryEmpty = [];
+
+export default {
+  title: '🛠 Email List/<cc-email-list>',
+  component: 'cc-email-list',
+};
+
+const conf = {
+  component: 'cc-email-list',
+};
+export const defaultStory = makeStory(conf, {
+  items: [
+    {
+      emails: {
+        state: 'loaded',
+        value: {
+          primaryAddress,
+          secondaryAddresses,
+        },
+      },
+    },
+  ],
+});
+
+export const skeleton = makeStory(conf, {
+  items: [
+    {},
```

Comment:
Should we be more explicit with:
```suggestion
    {
      emails: {
        state: 'loading',
      },
    },
```

(and same in `simulationsWithPrimary` & `simulationsWithSecondary`)